config/config.go

```diff
@@ -14,6 +14,8 @@ const (
 	ENV_APP_CONFIG_LOCATION_KEY  = "FLOGO_CONFIG_PATH"
 	APP_CONFIG_LOCATION_DEFAULT  = "flogo.json"
 	ENV_STOP_ENGINE_ON_ERROR_KEY = "FLOGO_ENGINE_STOP_ON_ERROR"
+	ENV_DATA_SECRET_KEY_KEY      = "FLOGO_DATA_SECRET_KEY"
+	ENV_DATA_SECRET_KEY_DEFAULT      = "flogo"
```

Comment:
should be just `DATA_SECRET_KEY_DEFAULT` rather than ENV_?

Reply:
It is just a constant name for env var following ENV_STOP_ENGINE_ON_ERROR_KEY. So, I feel its right.